packages/portal-proto/src/features/facets/NumericRangeFacet/FromTo.tsx

```diff
@@ -251,7 +253,7 @@ const FromTo: React.FC<FromToProps> = ({
             className="text-sm flex-1"
             placeholder={`Min: ${lowerUnitRange}${unitsLabel} `}
             // units are always days
-            value={form.values.fromValue ?? ""}
```

Comment:
I think these need to stay as empty strings or it will regress PEAR-1379: https://github.com/NCI-GDC/gdc-frontend-framework/pull/1198

Reply:
good catch, changed